llama_hub/web/sitemap/README.md

```diff
@@ -0,0 +1,36 @@
+# Sitemap Loader
+
+This loader is an asynchronous web scraper that fetches the text from static websites by using its sitemap and optionally converting the HTML to text.
+
+It is based on the [Async Website Loader](../async_web/README.md)
+
+## Usage
+
+To use this loader, you just declare the sitemap.xml url like this:
+
+```python
+from llama_index import download_loader
```

Comment:
slight nit: we changed default usage (llama-hub is a pypi package) so it should now be 

```python
from llama_hub.web.sitemap.base import AsyncWebPageReader
```

Reply:
done 👍🏻

Reply:
I updated the readme again, I think I had some errors in the example: 

https://github.com/emptycrown/llama-hub/pull/336